docs/source/customizations/coprocessor.mdx

```diff
@@ -825,6 +825,8 @@ When `stage` is `SupergraphResponse`, if present and `true` then there will be s
 
 An object mapping of all HTTP header names and values for the corresponding request or response.
 
+Headers should be handled like HTTP headers in general and normalized before your coprocessor operates on them. 
+
 If your coprocessor [returns a _different_ value](#responding-to-coprocessor-requests) for `headers`, the router replaces the existing headers with that value.
 ⚠️ `content-length` sent by Coprocessors are discarded by the router. (Wrong `content-length` values could lead to failure for HTTP requests to be sent)
```

Comment:
Might be worth cleaning this up a little too?

```suggestion
If your coprocessor [returns a _different_ value](#responding-to-coprocessor-requests) for `headers`, the router replaces the existing headers with that value.

> The router discards any `content-length` headers sent by coprocessors because incorrect `content-length` values can lead to HTTP request failures.
```

(Going with a blockquote rather than admonition because it might look a bit cluttered within a table cell.)